Source/SvgElement.cs

```diff
@@ -956,7 +956,20 @@ private bool TryResolveParentAttributeValue(string attributeKey, out object pare
             return resolved;
         }
 
-        public virtual void Write(XmlTextWriter writer)
+        /// <summary>
+        /// Write this SvgElement out using a given XmlWriter.
+        /// </summary>
+        /// <param name="writer">The XmlWriter to use.</param>
+        /// <remarks>
+        ///   Recommendation is to create an XmlWriter by calling a factory method,<br/>
+        ///   e.g. <see cref="XmlWriter.Create(System.IO.Stream)"/>,
+        ///   as per <a href="https://docs.microsoft.com/en-gb/dotnet/api/system.xml.xmltextreader?view=net-5.0#remarks">Microsoft documentation</a>.<br/>

```

Comment:
change URL to 
`https://docs.microsoft.com/dotnet/api/system.xml.xmltextreader#remarks`.

Reply:
changed

Reply:
Please delete language(`en-gb/`) as well.

Reply:
done